settings.py

```diff
@@ -46,3 +66,5 @@
     'es': '@'.join(['redaccion', 'cafebabel.com']),
     'it': '@'.join(['redazione', 'cafebabel.com']),
 }
+ARTICLES_IMAGES_URL = '/static/uploads/articles'
+ARTICLES_IMAGES_PATH = Path(__file__).parent / 'static/uploads/articles'
```

Comment:
I think the philosophy behind Path is to use `Path() / 'static' / 'blah'` and let Python deal with separators.